kong/runloop/plugins_iterator.lua

```diff
@@ -390,23 +422,63 @@ local function new_ws_data()
 end
 
 
+local function configure(configurable, ctx)
+  ctx = ctx or ngx.ctx
+  local kong_global = require "kong.global"
+  for _, plugin in ipairs(CONFIGURABLE_PLUGINS) do
+    local name = plugin.name
+
+    kong_global.set_namespaced_log(kong, plugin.name, ctx)
+    local start = utils.get_updated_monotonic_ms()
+    local ok, err = pcall(plugin.handler[CONFIGURE_PHASE], plugin.handler, configurable[name])
+    local elapsed = utils.get_updated_monotonic_ms() - start
+    kong_global.reset_log(kong, ctx)
+
+    if not ok then
+      kong.log.err("failed to execute plugin '", name, ":", CONFIGURE_PHASE, " (", err, ")")
+    else
+      if elapsed > 50 then
```

Comment:
any chance that elapsed can be `nil`?

Reply:
I don't think there is chance for that. We already use `get_monotonic_ms` and `get_updated_monotonic_ms` where we blindly trust that it works.